include/lsst/afw/table/Source.h

```diff
@@ -474,10 +474,10 @@ class SourceColumnViewT : public ColumnViewT<RecordT> {
     }
 
     ndarray::Array<double, 1> const getX() const {
-        return this->operator[](this->getTable()->getCentroidKey().getX());
+        return this->operator[](this->getTable()->getCentroidSlot().getMeasKey().getX());
```

Comment:
I'm not sure I understand why these changes are on their own commit, given that you remove other self-uses on the "Remove Deprecated Functions" commit. Could you rearrange the commits so that these come first? That way the code works at each step.

Reply:
Oh, sorry: I'd meant to squash these with the other commit, to make it all one block. I just forgot (I made incremental commits while I was fixing things).